julia.spdx.json

```
@@ -86,6 +86,18 @@
             "copyrightText": "Copyright (c) 2020 Stefan Karpinski <stefan@karpinski.org> and contributors",
             "summary": "ArgTools provides tools for creating consistent, flexible APIs that work with various kinds of function arguments."
         },
+        {
+            "name": "LinearAlgebra.jl",
+            "SPDXID": "SPDXRef-JuliaLinearAlgebra",
+            "downloadLocation": "git+https://github.com/Julia/LinearAlgera.jl.git",
```

there are some typos in here
```suggestion
            "downloadLocation": "git+https://github.com/JuliaLang/LinearAlgebra.jl.git",
```